ci/dra_aarch64.sh

```diff
@@ -9,26 +9,49 @@ export JRUBY_OPTS="-J-Xmx1g"
 # Extract the version number from the version.yml file
 # e.g.: 8.6.0
 # The suffix part like alpha1 etc is managed by the optional VERSION_QUALIFIER_OPT environment variable
-STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:\s\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
+STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:[[:space:]]\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
 
-if [ -z "$VERSION_QUALIFIER_OPT" ]; then
-  RELEASE=1 rake artifact:docker
-  RELEASE=1 rake artifact:docker_oss
-  rake artifact:dockerfiles
+# WORKFLOW_TYPE is a CI externally configured environment variable where, if not an empty string,
+# it's assumed to be snapshot
+if [ -z "$WORKFLOW_TYPE"]; then
```

Comment:
The comments said if not empty, it is a snaphot, but here it says `WORKFLOW_TYPE` is empty and append snapshot.
Do you want to check with `-n` for non empty?

Reply:
You are correct! it should be `-n`

Reply:
I've fixed it